### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by AnthonyBloomer**
This PR addresses two issues:

1. If we just run ```npm install``` we won't be able to run ```spodl spotify_account_name spotify_playlist_id```. The only way that will work is if we install globally using ```npm install -g```

2. If the user specifies the download directory in the config file, we still create a folder in the current directory with the playlist name so I've removed the function createDirectory. youtube-dl will handle that for us anyway.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,17 +19,12 @@ const dir = config.directory;
 const youTube = new YouTube();
 youTube.setKey(config.youtube.apikey);
 
-const createDirectory = (name) => {
-  playlistName = name;
-  if (!fs.existsSync(playlistName))
```
Comment (adhorrig) [inline src/index.js]: If this is removed what happens if a playlist is downloaded twice to the same directory? There is no check anymroe

**Comment by AnthonyBloomer** [comment]
If the directory doesn't exist then youtube-dl will create it. If it does exist then youtube-dl will download the playlist into that directory.

File: package.json [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
+	"name": "spodl",
+	"version": "1.0.0",
+	"description": "Spotify -> Local",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/anthonybloomer/spodl.git"
```
Comment (adhorrig) [inline package.json:11]: Wrong url

File: package.json [hunk 3]
```diff
@@ -1,29 +1,32 @@
...
+	"keywords": [
+		"spotify", "youtube", "youtube-dl"
+	],
+	"contributors": [
+		"Adam Horrigan",
+		"Anthony Bloomer"
+	],
+	"license": "MIT",
+	"bugs": {
+		"url": "https://github.com/anthonybloomer/spodl/issues"
```
Comment (adhorrig) [inline package.json:22]: Again

File: package.json [hunk 4]
```diff
@@ -1,29 +1,32 @@
...
+	},
+	"bin":{
+		"spodl": "src/index.js"
+	},
+	"homepage": "https://github.com/anthonybloomer/spodl#readme",
```
Comment (adhorrig) [inline package.json:27]: Again

File: config.js [hunk 5]
```diff
@@ -1,10 +0,0 @@
-module.exports = {
```
Comment (adhorrig) [inline config.js:1]: I'd rather handle the API keys this way than through ENV variables

File: src/index.js [hunk 6]
```diff
@@ -67,8 +60,8 @@ const yt = (searchTerm) => youTube.search(searchTerm, 1, ytCb);
 
 //Downloading
 const dl = (url) => {
-  const cmd = process.argv[4] === 'video' ? 'youtube-dl -o "' + dir + playlistName + '/%(title)s.%(ext)s"'  :
-  'youtube-dl -o "' + dir + playlistName + '/%(title)s.%(ext)s" --extract-audio --audio-format mp3 ';
+  const cmd = process.argv[4] === 'video' ? 'youtube-dl -o "' + playlistName + '/%(title)s.%(ext)s"'  :
```
Comment (adhorrig) [inline src/index.js:63]: Looks good 💯

**Review by adhorrig — CHANGES_REQUESTED** [review]
Readme update and creating of the directory looks good. However, I still want to use the config file for the API keys rather than environment variables.

Also try squash the commits again :)

**Pull request closed without merging**